packages/pwa-kit-react-sdk/package.json

```diff
@@ -10,7 +10,8 @@
     "sdk-create-hash-manifest": "bin/create-hash-manifest.js"
   },
   "engines": {
-    "node": "^12.0.0 || ^14.0.0"
+    "node": "^12.0.0 || ^14.0.0",
+    "npm": "^6.14.4 || ^7.0.0 || ^8.0.0"
```

Comment:
Making sure that the `engines` value in the root's package.json is copied into other packages' too.